manual/README.md

```diff
@@ -146,3 +148,42 @@ The following setup assumes a Unix-like environment.
     <br/><br/>
 
 12. Apply to PoA. Please contact the Mezo team for further instructions.
+
+## Runbooks
+
+### State sync from snapshot
+
+To start a node from a snapshot is no different to any other CometBFT based blockchain.
+
+To start with, select a block from a trusted node. You can do so using the following API:
+`http://<NODE_ADDRESS>:26657/block`, this will give you the very last block
+executed by the node.
+
+Once done, you will need to update the following fields in the CometBFT config. You can find this
+config in the `MEZOD_HOME/config/config.toml` file, under the `statesync` TOML section:
+- `enable`: should be set to `true` to enable state sync (can be flipped back once the node synchronizes)
+- `rpc_servers`: is a list of trusted node RPC servers that the node can use to recover blocks from
+- `trust_height`: should be the height of the selected trusted block
+- `trust_hash`:  should be the hash of the selected trusted block
+
+Moreover, please make sure your node doesn't have any local state while using state sync.
+
+Here is an example configuration you can use for the Mezo Matsnet testnet. Be sure to
+update the `trust_height` and `trust_hash` fields as explained previously. Everything
+else would be valid as-is:
+
+```toml
```

Comment:
Let's fix formatting of this code block:
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/30b29b6a-a19f-451a-b608-80436e1ccc21" />